docs/indyav/index.rst

```diff
@@ -0,0 +1,30 @@
+Indy AV
+=====================
+Welcome to team Gator Double Dragon's documentation. The team is a collabortaion
+between the University of Florida and Kookmin University for the
+`Indy Autonomous Challenge <https://www.indyautonomouschallenge.com/>`_.
+
+.. image:: gator_double_dragon.jpg
+   :width: 600px
+   :align: center
+   :height: 420px
+
+Collaborators
```

Comment:
`Collaborators` -> `Collaborating Laboratories`

Reply:
I don't believe this change provides any benefit